object_database/web/cells.py

```diff
@@ -885,28 +906,30 @@ def __init__(self, body, header=None, padding=None):
     def recalculate(self):
         self.children = {"____contents__": Cell.makeCell(self.body)}
 
-        if self.header is not None:
-            self.children['____header__'] = Cell.makeCell(self.header)
-
         other = ""
         if self.padding:
             other += " p-" + str(self.padding)
 
-        self.contents = ("""
-            <div class="card" __style__>
-              __header__
-              <div class="card-body __other__">
-                ____contents__
-              </div>
-            </div>
-            """.replace('__other__', other)
-               .replace('__style__', self._divStyle())
-               .replace("__header__", """
-                    <div class="card-header">
-                        ____header__
-                    </div>
-                """ if self.header is not None else "")
-        )
+        body = HTMLElement.div().add_class(
+            "card-body").add_class(
+                other).add_child(
+                    HTMLTextContent(" ____contents__"))
```

Comment:
I also find this easier to read

Reply:
Yeah. In fact, the format you've proposed @atzannes is what would normally be used in JS. I just haven't encountered "chainable" patterns in Python too often and wasn't sure how to format them properly and have linters pass. I believe you would need a `\` at the end of each line in that construction though, right? Either way the "separate lines" approach is certainly easier to read.

Reply:
If I'm not mistaken, we do not need the `\` at the end of the line because we are within a parenthesized expression. Both the linter and python should be happy with that syntax